packages/lottie/src/Lottie.tsx

```diff
@@ -0,0 +1,108 @@
+import type {AnimationItem} from 'lottie-web';
+import lottie from 'lottie-web';
+import type {CSSProperties} from 'react';
+import {useEffect, useRef, useState} from 'react';
+import {continueRender, delayRender, useCurrentFrame} from 'remotion';
+import type {LottieAnimationData} from './types';
+import {getNextFrame} from './utils';
+import {validateLoop} from './validate-loop';
+import {validatePlaybackRate} from './validate-playbackrate';
+
+export interface LottieProps {
+	/**
+	 * JSON object with the animation data.
+	 * */
+	animationData: LottieAnimationData;
+	/**
+	 * If the animation should loop after its end.
+	 */
+	loop?: boolean;
+	/**
+	 * The speed of the animation. Defaults to 1.
+	 */
+	playbackRate?: number;
+	/**
+	 * CSS classes to apply on the container of the animation.
+	 */
+	className?: string;
+	/**
+	 * CSS properties to apply on the container of the animation.
+	 */
+	style?: CSSProperties;
+}
+
+export const Lottie = ({
+	animationData,
+	className,
+	loop,
+	playbackRate,
+	style,
+}: LottieProps) => {
+	if (typeof animationData !== 'object') {
+		throw new Error(
+			'animationData should be provided as an object. If you only have the path to the JSON file, load it and pass it as animationData. See https://remotion.dev/link-tbd for more information.'
```

Comment:
We need to update this error message. I put `https://remotion.dev/link-tbd` as a placeholder, which page should we link it to?

Reply:
Thanks! I pointed it to the Lottie example snippet and added links for all 3 ways to do it